src/sass/static/pages.scss

```diff
@@ -1708,3 +1708,14 @@ body #not_authenticated_financial ul.checked > li {
 #self_exclusion_dialog #dialog_content {
     max-width: 496px;
 }
+
+.dropdown__icon-filter {
+    position: relative;
+    left: 30px;
+    top: -2px;
+    pointer-events: none;
+}
+
+.dropdown-statement-filter {
```

Comment:
Any reason why you use both identical id and classname to refer to this component?

Reply:
So, why do we need both class and id for this?